tracing-subscriber/Cargo.toml

```diff
@@ -52,6 +51,9 @@ tracing-serde = { path = "../tracing-serde", optional = true }
 # opt-in deps
 parking_lot = { version = ">= 0.7, < 0.10", features = ["owning_ref"], optional = true }
```

Comment:
owning_ref feature is no longer required
```suggestion
parking_lot = { version = ">= 0.7, < 0.10", optional = true }
```

Reply:
fixed!